dotcom-rendering/src/lib/branding.test.ts

```diff
@@ -629,3 +629,35 @@ describe('decideCollectionBranding', () => {
 		expect(collectionBranding).toBeUndefined();
 	});
 });
+
+describe('decideTagPageBranding', () => {
+	it('picks branding from a tag page by their edition', () => {
+		const branding = {
+			brandingType: { name: 'sponsored' as const },
+			sponsorName: 'Guardian.org',
+			aboutThisLink: '',
+			logo,
+		};
```

Comment:
You can also use the [`satisfies` operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-9.html#the-satisfies-operator) here instead of only making `'sponsored' as const`:

```suggestion
		const branding = {
			brandingType: { name: 'sponsored' },
			sponsorName: 'Guardian.org',
			aboutThisLink: '',
			logo,
		} satisfies Branding;
```

Reply:
Ah great, thanks for the tip!